README.md

```diff
@@ -152,11 +152,13 @@ function updateZoomIngester() {
 1. Install the app in the workspace.
 1. Add the following environment variables to your `.env` file:
 
+    * On the Permissions page. Copy/paste the "Bot User OAuth Token" into your .env file `SLACK_API_TOKEN`.
```

Comment:
Makes sense to move this just because of what page you end up on right after you've installed the app into your workspace.